loaders/src/glTF/2.0/Extensions/KHR_materials_transmission.ts

```diff
@@ -6,6 +6,258 @@ import { IMaterial } from "../glTFLoaderInterfaces";
 import { IGLTFLoaderExtension } from "../glTFLoaderExtension";
 import { GLTFLoader } from "../glTFLoader";
 import { IKHRMaterialsTransmission } from 'babylonjs-gltf2interface';
+import { Scene } from "babylonjs/scene";
+import { AbstractMesh } from "babylonjs/Meshes/abstractMesh";
+import { Mesh } from "babylonjs/Meshes/mesh";
+import { Texture } from "babylonjs/Materials/Textures/texture";
+import { RenderTargetTexture } from "babylonjs/Materials/Textures/renderTargetTexture";
+import { Observable } from "babylonjs/Misc/observable";
+
+interface ITransmissionHelperOptions {
+    /**
+     * The size of the render buffers
+     */
+    renderSize: number;
+}
+
+/**
+ *
+ */
+export class TransmissionHelper {
+
+    /**
+     * Creates the default options for the helper.
+     */
+    private static _getDefaultOptions(): ITransmissionHelperOptions {
+        return {
+            renderSize: 512
+        };
+    }
+
+    /**
+     * Stores the creation options.
+     */
+    private readonly _scene: Scene;
+    private _options: ITransmissionHelperOptions;
+    private _opaqueRTIndex: number = -1;
+
+    private _opaqueRenderTarget: Nullable<RenderTargetTexture> = null;
+    private _opaqueMeshesCache: Mesh[] = [];
+    private _transparentMeshesCache: Mesh[] = [];
+
+    public disabled: boolean = false;
```

Comment:
nit: Double negative for properties are confusing. Consider using `enabled` instead.

Reply:
I'm also a bit confused by this property being public. Are you expecting users of this class to be able to set this?

Reply:
Actually, is this property even necessary? It seems to be only used in updateOptions. Is that expected?

Reply:
Yeah, sorry. This is left over from the much more complex version that I have in the Adobe viewer. I'll try to clean this up in this version.

Reply:
Not sure if you meant to remove the property or still keep it?

Reply:
Sorry, I should remove it. Thanks.